.github/workflows/release.yml

```diff
@@ -153,7 +153,7 @@ jobs:
         with:
           repository: pentaho/actions-common
           path: actions-common
-          ref: stable
+          ref: HNC-573
```

Comment:
This should continue to be `stable`, right?

Reply:
yes it should be stable so that we merge it.